UKHO.MaritimeSafetyInformation.Web/UKHO.MaritimeSafetyInformation.Common.UnitTests/Helpers/NMHelperTest.cs

```diff
@@ -0,0 +1,425 @@
+                            FileDescription = "Daily 23-04-22.zip",
+                            FileExtension = ".zip",
+                            FileSizeinKB = FileHelper.FormatSize(346040),
+                            MimeType = "application/gzip",
+                            Links = null,
+                            AllFilesZipSize = 346040
+                        }
+                    }
+                }
+            };
+            expected = expected.OrderByDescending(x => x.Year).ThenByDescending(x => x.WeekNumber).ToList();
+
+            foreach (var item in expected)
+            {
+                item.DailyFilesData = item.DailyFilesData.OrderBy(x => Convert.ToDateTime(x.DataDate)).ToList();
+            }
+
+            List<ShowDailyFilesResponseModel> result = NMHelper.GetDailyShowFilesResponse(SearchResult);
+            Assert.Multiple(() =>
+            {
+                for (int i = 0; i < result.Count; i++)
+                {
+                    Assert.AreEqual(expected[i].Year, result[i].Year);
+                    Assert.AreEqual(expected[i].YearWeek, result[i].YearWeek);
+                    Assert.AreEqual(expected[i].WeekNumber, result[i].WeekNumber);
+
+                    for (int j = 0; j < expected[i].DailyFilesData.Count; j++)
+                    {
+                        Assert.AreEqual(expected[i].DailyFilesData[j].BatchId, result[i].DailyFilesData[j].BatchId);
+                        Assert.AreEqual(expected[i].DailyFilesData[j].DataDate, result[i].DailyFilesData[j].DataDate);
+                        Assert.AreEqual(expected[i].DailyFilesData[j].Filename, result[i].DailyFilesData[j].Filename);
+                        Assert.AreEqual(expected[i].DailyFilesData[j].FileDescription, result[i].DailyFilesData[j].FileDescription);
+                        Assert.AreEqual(expected[i].DailyFilesData[j].FileExtension, result[i].DailyFilesData[j].FileExtension);
+                        Assert.AreEqual(expected[i].DailyFilesData[j].FileSizeinKB, result[i].DailyFilesData[j].FileSizeinKB);
+                        Assert.AreEqual(expected[i].DailyFilesData[j].MimeType, result[i].DailyFilesData[j].MimeType);
+                        Assert.AreEqual(expected[i].DailyFilesData[j].Links, result[i].DailyFilesData[j].Links);
+                        Assert.AreEqual(expected[i].DailyFilesData[j].AllFilesZipSize, result[i].DailyFilesData[j].AllFilesZipSize);
+                    }
+                }
+            });
+        }
+
+        [Test]
+        public void WhenGetFormattedDateIsCalled_ThenShouldReturnDateInExpectedFormat()
+        {
+            string strdate = "2022-05-26";
+            string actualDate = NMHelper.GetFormattedDate(strdate);
+            string expectedDate = "26-05-22";
+            Assert.AreEqual(expectedDate, actualDate);
+        }
+
+        private static BatchSearchResponse SetSearchResultForWeekly()
+        {
+            BatchSearchResponse SearchResult = new()
```

Comment:
Camel case variable names please:
```suggestion
            BatchSearchResponse searchResult = new()
```

Reply:
Camel case done